src/javascript/expression.rs

```diff
@@ -464,6 +472,26 @@ fn array<'a, Elements: Iterator<Item = Output<'a>>>(elements: Elements) -> Outpu
     .group())
 }
 
+fn wrap_list<'a, E>(
+    elements: &'a [E],
+    inner: Document<'a>,
+    mut renderer: impl FnMut(&'a E) -> Output<'a>,
```

Comment:
needs to implement FnMut because the closure in the expression has a reference to self.